addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animations.py

```diff
@@ -258,6 +259,34 @@ def __get_blender_actions(blender_object: bpy.types.Object,
                         blender_tracks[strip.action.name] = track.name # Always set after possible active action -> None will be overwrite
                         action_on_type[strip.action.name] = "SHAPEKEY"
 
+    # Material animation - not actually used by the base exporter, but useful for addons that take advantage of it.
```

Comment:
I don't like so much to have "dead code" that is not used by the vanilla exporter. Maybe we can add a hook here to be able to manage this kind on export data, where you can retrieve these needed information for your extension?

Reply:
Yeah, you make a great point. I just updated the PR to switch to utilizing hooks. One question though, on the `gather_actions` hook I added, if the add-on modifies the `blender_action`, `blender_tracks`, or `action_on_type` parameters, would it change those values in the export?